scalafmt-sbt-tests/src/sbt-test/scalafmt-sbt/sbt/test

```diff
@@ -0,0 +1,24 @@
+> p123/compile:scalafmt
```

Comment:
Should we maybe add a negative check here?

```
-> p123/compile:scalafmtCheck
```

to validate check fails on misformatted files.